dind/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM alpine:3.5
 
-ENV DOCKER_VERSION=1.13.1 \
-  DOCKER_COMPOSE_VERSION=1.20.1 \
+ENV DOCKER_VERSION=18.09.9 \
+  DOCKER_COMPOSE_VERSION=1.22.0 \
   DOCKER_PACKAGES="curl device-mapper mkinitfs bash e2fsprogs e2fsprogs-extra iptables python-dev" \
   OTHER="git"
 
 # Install Docker, Docker Compose
 RUN apk --update --no-cache add $DOCKER_PACKAGES $OTHER && \
-    curl https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION}.tgz | tar zx && \
+    curl https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz | tar zx && \
     mv /docker/* /bin/ && chmod +x /bin/docker* \
 && \
     apk add py-pip && \
-    pip install docker-compose==${DOCKER_COMPOSE_VERSION}
+    pip install docker-compose==${DOCKER_COMPOSE_VERSION} \
+&& \
+    docker -v && \
+    docker-compose -v
```

Comment:
want to display the version of docker and docker-compose as a record in the build